frontend/src/views/WheresMyGene/components/GeneSearchBar/components/Organism/index.tsx

```diff
@@ -77,6 +77,7 @@ export default function Organism({ isLoading }: Props): JSX.Element {
         onChange={handleOnChange as tempOnChange}
         InputDropdownProps={{ ...InputDropdownProps, disabled: isLoading }}
         data-test-id="add-organism"
+        value={organism}
```

Comment:
need to add `value` prop to show active state on the default value

Reply:
Great question! Those two filters are from `QuickSelect/index.tsx`, which under the hood uses SDS `MenuSelect` and we are already passing `value` prop to `MenuSelect`!

https://github.com/chanzuckerberg/single-cell-data-portal/blob/73b444ac8f37a85f4c41e4e6b49351580ce82e93/frontend/src/views/WheresMyGene/components/GeneSearchBar/components/QuickSelect/index.tsx#L298

Reply:
Oh wait, I misread your message 😆 You meant why CellTypes and Genes sort by options don't need `value`, not the QuickSelect for Genes and Tissues

I looked into this more. I think essentially hard coded options don't need `value` because the object reference stay the same, whereas `organisms` we filter them with `useMemo()`, so we need to pass `value` prop OR add the following as prop `DropdownMenuProps`:

```ts
const DropdownMenuProps = {
  isOptionEqualToValue: (
    option: DefaultMenuSelectOption,
    value: DefaultMenuSelectOption
  ) => option.name === value.name,
};
```

I decided to go with `DropdownMenuProps` since I think it's more intuitive to use!

Reply:
Actually I just found out that without `value`, it won't highlight the default option when you open the dropdown for the first time, so I changed back to using `value`

I tried added `value` to sortBy for genes/cell types and somehow it was causing an infinite loop, so will need to look into that later!

Reply:
just added `value` to sortBy genes/cell types too to make default active work too. Turned out it was just my Compare branch that was buggy 😆